content/04.results.md

```diff
@@ -70,15 +71,15 @@ Preprints from certain topic areas that spanned approaches from informatics-rela
 **B.** Potential preprint-publication pairs that are unannotated but within the 50th percentile of all preprint-publication pairs in the document embedding space are likely to represent true preprint-publication pairs.
 We depict the fraction of true positives over the total number of pairs in each bin.
 Accuracy is derived from the curation of a randomized list of 200 potential pairs (50 per quantile) performed in duplicate with a third rater used in the case of disagreement.
-**C.** Most preprints are eventually published.
+**C.** Most preprints are eventually published even when adjusting for preprints posted in 2019 and on.
 We show the publication rate of preprints since bioRxiv first started.
 The x-axis represents months since bioRxiv started, and the y-axis represents the proportion of preprints published given the month they were posted.
 The light blue line represents the publication rate previously estimated by Abdill et al. [@doi:10.7554/eLife.45133].
```

Comment:
Is this the publication rate that they estimated or one that you derived from their estimates?